src/Sylius/Bundle/ProductBundle/Behat/ProductContext.php

```diff
@@ -292,4 +292,35 @@ public function theFollowingOptionTranslationsExist(TableNode $table)
 
         $manager->flush();
     }
+
+    /**
+     * @Given there is a disabled product :name
+     */
+    public function thereIsDisabledProduct($name)
+    {
+        $this->thereIsProduct($name, false);
+    }
+
+    /**
+     * @Given there is an enabled product :name
+     */
+    public function thereIsEnabledProduct($name)
+    {
+        $this->thereIsProduct($name, true);
+    }
+
+    /**
+     * @Given /^There is a product named "([^""]*)"$/
```

Comment:
I would suggest take a look at https://github.com/Sylius/Sylius/blob/master/src/Sylius/Bundle/LocaleBundle/Behat/LocaleContext.php#L23. 

The enhancement should be either extract object creation to a separate method or swap condition, if not null then return else create object outside `if` brackets.